File: expressions/budget.c
#include <stdio.h>

int main(void){
  float income, rent, untilities, grocceries, transportation, savings, expenses, spend;
  float prent, putilities, pgrocceries, ptransportion, psavings, pexpenses;
  printf("This is a budget calculator.\n How much do you make a month?\n");
  scanf("%f", &income); 
  printf("How much does your utilities cost?\n");
  scanf("%f", &untilities); 
  printf("How much does your groceries cost?\n");
  scanf("%f", &grocceries);   
  printf("How much does your transportation cost?\n");
  scanf("%f", &transportation); 
savings = income * .2;
expenses = rent + untilities + grocceries + transportation + savings;
 spend = income - expenses - savings;
 printf("You make $%.2f\n", income);
  printf("You make $%.2f\n", expenses);
   printf("You make $%.2f\n", savings);
    printf("You spending money is $%.2f\n", spend);
    prent = rent/income * 100;
    putilities = untilities/income * 100;
    pgrocceries = grocceries/income * 100;
    ptransportion =transportation/income * 100;
    psavings = savings/income * 100;
    pexpenses = expenses/income * 100; 
    printf("Your rent is %d%% of your income.\n", (int) prent);
        printf("Your utilities is %d%% of your income.\n", (int) putilities);
          printf("Your groceries is %d%% of your income.\n", (int) pgrocceries);
          printf("Your transportation is %d%% of your income. \n", (int)ptransportion); 
       printf("Your savings are %d%% of your income. \n", (int)psavings); 
       printf("Your expenses are %d%% of your income. \n",(int)pexpenses);
  return 0;
}

File: expressions/budget.py
print("This is a finacial calculator")
income = input("how much do you make a month?:\n")
rent = input("how much is your rent?:\n")
utilitties=input("How much is your utilitties?:\n")
groceries=input("How much do your goceries cost?:\n")
transp=input("What does your transportation cost?:\n")
print("You make", income)
savings = float(income)*float(0.2)
expenses = (float(rent) + float(utilitties) + float(groceries) + float(transp))
spending = float(income)-float(expenses)-float(savings)
print("your expenses are", expenses)
print("your savings is", savings)
print("your spending money is", spending)
prent=(float(rent)/float(income))*100 
putilitties=(float(utilitties)/float(income))*100
pgroceries=(float(groceries)/float(income))*100
ptransp=(float(transp)/float(income))*100
print("your rent takes up this percent of your income;", prent)
print("your utilitties take up this percent of your income;", putilitties)
print("your groceries take up this percent of your income;", pgroceries)
print("you transportation takes up this percent of your income;", ptransp)